test/test_tensor_creation_ops.py

```diff
@@ -626,16 +626,22 @@ def test_cat_out(self, device):
         y = torch.randn((4, 6), device=device)
 
         with self.assertRaisesRegex(
-                RuntimeError, r"unsupported operation:.* input tensor 0"):
+                RuntimeError,
+                r"unsupported operation: some elements of the input tensor and "
```

Comment:
Error messages changed here, because I'm using `at::assert_no_overlap` function. Should I revert that to show better error messages?

Reply:
This seems reasonable. The previous error message was also related to overlap:
```
>>> torch.cat([x, y], out=x)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: 0unsupported operation: the input tensors cannot refer to any of the output memory locations. Found overlap in input tensor 0
```

Reply:
Yeah. I was a bit worried that not having the number of the input tensor that had an overlap would be bad. :)